rpmlb/__init__.py

```diff
@@ -1,25 +1 @@
-"""Top-level module
-"""
-
-import logging
-
 __version__ = '1.0.0'
```

Comment:
ok, thanks for the information. and welcome back :) I agree with you.

> As a side note, what about setuptools_scm? It seems to be more convenient that tracking the version in a string somewhere…

I am not sure about it.
So, if you like, I prefer you would send another PR for that after this PR.
Maybe @hroncok will accept it.